x-pack/plugins/security_solution/public/common/store/sourcerer/helpers.ts

```diff
@@ -6,25 +6,28 @@
  */
 
 import { isEmpty } from 'lodash';
-import { SourcererModel, SourcererScopeName } from './model';
-import { TimelineEventsType } from '../../../../common/types/timeline';
+import { KibanaIndexPattern, SourcererModel, SourcererScopeName } from './model';
+import { TimelineEventsType } from '../../../../common';
 
 export interface Args {
   eventType?: TimelineEventsType;
   id: SourcererScopeName;
   selectedPatterns: string[];
   state: SourcererModel;
 }
+
+export const getPatternList = (kip: KibanaIndexPattern): string[] => [kip.title]; // kip.title.split(','); // TODO: Steph/sourcerer implement splitting KIPs
```

Comment:
TODO, did you want to take care of this in this PR or in a follow up?

Reply:
oh boy oh boy, wait until you see the next PR ;P 

but yes a follow up will take care of it